doc/user/content/integrations/cli/reference/app-password.md

```diff
@@ -0,0 +1,65 @@
+---
+title: mz app-password
+description: The `mz app-password` command manages app passwords for your user account.
+menu:
+  main:
+    parent: cli-reference
+    weight: 1
+---
+
+The `mz app-password` command manages app passwords for your user account.
```

Comment:
What if we use `mz password`? Every time I want to use this command my mind defaults to type `mz password`. The `app-` feels informative but counter-intuitive

Reply:
Unfortunately that results in a _different_ problem, which is conflation with the password you log in to the web UI. "App password" was the shortest term we could come up with to distinguish between the two types of passwords.